charts/ctlog/values.yaml

```diff
@@ -6,7 +6,6 @@ server:
   replicaCount: 1
   config:
     secret:
-      enabled: false
       name: ctlog-config
```

Comment:
can we delete this as well, it does not seem to be used anywhere, which means the entire `secret` can be deleted